README.md

```diff
@@ -4,61 +4,180 @@ Mitschriften und Formelsammlungen in LaTeX für Elektro- und Informationstechnik
 
 <!-- mdformat-toc start --slug=github --no-anchors --maxlevel=6 --minlevel=2 -->
 
-- [Mitarbeiten](#mitarbeiten)
-- [Formelsammungen](#formelsammungen)
-  - [Allgemein](#allgemein)
-  - [1. Semester](#1-semester)
-  - [2. Semester](#2-semester)
-  - [3. Semester](#3-semester)
-  - [4. Semester](#4-semester)
-  - [Externe Formelsammlungen](#externe-formelsammlungen)
-- [Formelsammlungen für Latex4ei](#formelsammlungen-f%C3%BCr-latex4ei)
-- [Optional: Build with CMake](#optional-build-with-cmake)
-  - [Steps to build](#steps-to-build)
+## 📚 Table of Contents
```

Comment:
Move this line above of line 5, then we can use the pre-commit hook to keep the toc down here automatically up to date